### PR TITLE
Fix header

### PR DESCRIPTION
**Pull request by WilliamTraoreee**
# What does this PR do?

Fix some UI elements on the header : 

- Skeleton position on status menu
- Hide the elements who don't retrieve data at this moment 

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by RemiBonnet** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Fix header - 2022-05-12T09:36:46Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/9f1b6cd1-4f83-4d6d-9171-48af8a7a108e/applications)

**Review by bdebon — APPROVED** [review]
lgtm

File: libs/pages/applications/ui/src/lib/components/container/container.tsx [hunk 1]
```diff
@@ -31,32 +32,30 @@ export function Container(props: ContainerProps) {
...
   )
 
-  const clickAction = (e: ClickEvent, action: string) => {
-    console.log(e)
-  }
-
   const headerActions = (
     <>
-      <Skeleton width={150} height={24} show={environment?.status ? false : true}>
+      <Skeleton width={150} height={24} show={!environment?.status} className={`${!environment?.status && 'mt-[1px]'}`}>
```
Comment (RemiBonnet) [inline libs/pages/applications/ui/src/lib/components/container/container.tsx:44]: ClassName, can't have && condition if you have undefined on the html

File: libs/pages/application/ui/src/lib/components/container/container.tsx [hunk 2]
```diff
@@ -31,31 +33,38 @@ export function Container(props: ContainerProps) {
...
   )
 
   const headerActions = (
     <>
-      <Skeleton width={150} height={24} show={application?.status ? false : true}>
+      <Skeleton
+        width={150}
+        height={24}
+        show={application?.status ? false : true}
+        className={`${!application?.status && 'mt-[1px]'}`}
```
Comment (RemiBonnet) [inline libs/pages/application/ui/src/lib/components/container/container.tsx:52]: same here

**Pull request merged**